network-api/networkapi/wagtailpages/pagemodels/customblocks/image_feature_block.py

```diff
@@ -0,0 +1,24 @@
+from wagtail.core import blocks
+from wagtail.images.blocks import ImageChooserBlock
+
+
+class ImageFeatureBlock(blocks.StructBlock):
+    image = ImageChooserBlock()
+
+    altText = blocks.CharBlock(
+        required=True,
```

Comment:
Do we need this? I think `required=True` is the default.

Reply:
removed